server/src/test/scala/io/delta/sharing/server/DeltaSharingServiceSuite.scala

```diff
@@ -1410,6 +1414,165 @@ class DeltaSharingServiceSuite extends FunSuite with BeforeAndAfterAll {
+    val protocol = lines(0)
+    val metadata = lines(1)
+    val expectedProtocol = DeltaFormatResponseProtocol(deltaProtocol = DeltaProtocol(minReaderVersion = 2, minWriterVersion = 5)).wrap
+
+    assert(expectedProtocol == JsonUtils.fromJson[DeltaResponseSingleAction](protocol))
+
+    val expectedConfigMap: Map[String, String] = List(
+      "delta.enableChangeDataFeed" -> "true",
+      "delta.enableDeletionVectors" -> "false",
+      "delta.columnMapping.maxColumnId" -> "2",
+      "delta.columnMapping.mode" -> "name").toMap
+
+    val expectedMetadata = DeltaResponseMetadata(
+      deltaMetadata = DeltaMetadataCopy(
+        id = "99b3ffc7-9852-4a29-9741-bfe54a326a4a",
+        name = null,
+        description = null,
+        format = DeltaFormat(
+          provider = RESPONSE_FORMAT_PARQUET,
+          options = Map()
+        ),
+        schemaString = """{"type":"struct","fields":[{"name":"date","type":"date","nullable":true,"metadata":{"delta.columnMapping.id":1,"delta.columnMapping.physicalName":"col-8bd94026-1115-4d95-aa45-f936cfc5fba3"}},{"name":"eventTime","type":"timestamp","nullable":true,"metadata":{"delta.columnMapping.id":2,"delta.columnMapping.physicalName":"col-6af3f748-93b3-44a0-b0ec-167be74ef613"}}]}""",
+        partitionColumns = Seq("date"),
+        configuration = expectedConfigMap,
+        createdTime = Option(1719346253102L)
+      )
+    ).wrap
+    assert(expectedMetadata == JsonUtils.fromJson[DeltaResponseSingleAction](metadata))
+
+    val files = lines.drop(2)
+    val actualFiles = files.map(f => JsonUtils.fromJson[DeltaResponseSingleAction](f).file)
+    assert(actualFiles.size == 1)
+
+    val expectedPartitionMap: Map[String, String] = List(
+      "col-8bd94026-1115-4d95-aa45-f936cfc5fba3" -> "2024-06-25").toMap
+    val expectedFiles = Seq(
+      DeltaResponseFileAction(
+        id = "fae5d45962a2617c129aabbbb0313d7f96ffc0d3a5b60fa5e02993c01cd02715",
+        version = null,
+        timestamp = null,
+        expirationTimestamp = actualFiles(0).expirationTimestamp,
+        deltaSingleAction = DeltaSingleAction(
+          add = DeltaAddFile(
+            path = actualFiles(0).deltaSingleAction.add.path,
+            partitionValues = expectedPartitionMap,
+            size = 688,
+            modificationTime = 1719346303000L,
+            dataChange = true,
+            stats = """{"numRecords":3,"minValues":{"col-6af3f748-93b3-44a0-b0ec-167be74ef613":"2024-06-25T00:00:00.000Z"},"maxValues":{"col-6af3f748-93b3-44a0-b0ec-167be74ef613":"2024-06-25T01:00:00.000Z"},"nullCount":{"col-6af3f748-93b3-44a0-b0ec-167be74ef613":0}}"""
+          )
+        )
+      )
+    )
+    assert(expectedFiles == actualFiles.toList)
+    assert(actualFiles.count(_.expirationTimestamp != null) == 1)
+    verifyPreSignedUrl(actualFiles(0).deltaSingleAction.add.path, 688)
+  }
+
+  integrationTest("column mapping query test 2 - /shares/{share}/schemas/{schema}/tables/{table}/metadata") {
```

Comment:
could you update the test names to match the actual query you are using? `/metadata` -> `/query`

Reply:
fixed